java/api/src/main/java/io/ray/api/options/PlacementGroupCreationOptions.java

```diff
@@ -0,0 +1,88 @@
+package io.ray.api.options;
+
+import io.ray.api.Ray;
+import io.ray.api.placementgroup.PlacementStrategy;
+import java.util.List;
+import java.util.Map;
+
+/** The options for creating placement group. */
+public class PlacementGroupCreationOptions {
+  public final boolean global;
+  public final String name;
+  public final List<Map<String, Double>> bundles;
+  public final PlacementStrategy strategy;
+
+  public PlacementGroupCreationOptions(
+      boolean global, String name, List<Map<String, Double>> bundles, PlacementStrategy strategy) {
+    this.global = global;
+    this.name = name;
+    this.bundles = bundles;
+    this.strategy = strategy;
+  }
+
+  /** The inner class for building PlacementGroupCreationOptions. */
+  public static class Builder {
+    private boolean global;
+    private String name;
+    private List<Map<String, Double>> bundles;
+    private PlacementStrategy strategy;
+
+    /**
+     * Set the name of a named placement group. This named placement group is only accessible from
+     * this job by this name via {@link Ray#getPlacementGroup(java.lang.String)}. If you want create
+     * a named placement group that is accessible from all jobs, use {@link
+     * Builder#setGlobalName(java.lang.String)} instead.
+     *
+     * @param name The name of the named placement group.
+     * @return self
+     */
+    public Builder setName(String name) {
+      this.name = name;
+      this.global = false;
+      return this;
+    }
+
+    /**
+     * Set the name of a named placement group. This placement group will be accessible from all
+     * jobs by this name via {@link Ray#getGlobalPlacementGroup(java.lang.String)}. If you want to
+     * create a named placement group that is only accessible from this job, use {@link
+     * Builder#setName(java.lang.String)} instead.
+     *
+     * @param name The name of the named placement group.
+     * @return self
+     */
+    public Builder setGlobalName(String name) {
+      this.name = name;
```

Comment:
Check this.name is null?

Reply:
yeah, that's a good idea.

Reply:
About the API naming, I think using `setName` with overloads is a better design. e.g.

```
public Builder setName(String name, boolean global) {
	this.name = name;
	this.global = global;
}

public Builder setName(String name) {
	return setName(name, /*global=*/false);
}
```

Reply:
yeah, that's a good idea @kfstorm